src/components/themeProvider/index.tsx

```diff
@@ -3,24 +3,21 @@ import { ChakraProvider } from '@chakra-ui/react';
 
 import { Brand } from 'src/types/brand';
 
-import { autoScout24Theme, motoScout24Theme } from 'src/themes';
+import { autoScout24System, motoScout24System } from 'src/themes';
 
 export type Props = {
   // Theme to use
   theme: Brand;
 };
 
 const themes = {
-  [Brand.AutoScout24]: autoScout24Theme,
-  [Brand.MotoScout24]: motoScout24Theme,
+  [Brand.AutoScout24]: autoScout24System,
+  [Brand.MotoScout24]: motoScout24System,
 };
 
-const ThemeProvider: FC<PropsWithChildren<Props>> = ({ theme, children }) => {
-  return (
-    <ChakraProvider theme={themes[theme]} resetCSS={true}>
```

Comment:
There isn't a global CSS reset provided by chakra anymore. I opted against porting what they provided in v2 and went with a minimal set of global styles. We very likely won't need it anymore as modern browsers don't overload on weird user-agent styles. 
If it's something we need (we need to check it during the migration), we can leverage something like [normalize.css](https://github.com/necolas/normalize.css)